www/class/centreon-knowledge/procedures_Proxy.class.php

```diff
@@ -64,21 +61,11 @@ public function __construct($pearDB, $host_name, $service_description = null)
 
         $conf = getWikiConfig($this->DB);
         $this->wikiUrl = $conf['kb_wiki_url'];
-        $this->proc = new procedures(
-            $this->DB
-        );
-
-        if (isset($host_name)) {
-            if (isset($service_description)) {
-                $this->returnServiceWikiUrl($this->DB->escape($host_name), $this->DB->escape($service_description));
-            } else {
-                $this->returnHostWikiUrl($this->DB->escape($host_name));
-            }
-        }
+        $this->proc = new procedures($this->DB);
```

Comment:
Do not create an instance in the constructor but rather pass it through the constructor parameters if it's possible

Reply:
this is just coding style here, I did non modify anything.
Side effects will be hard to fix if I update this